os-appium-tests/config/wdio.shared.conf.js

```diff
@@ -0,0 +1,76 @@
+exports.config = {
+    maxInstances: 1,
+    // ============
+    // Specs
+    // ============
+    specs: [
+        './tests/specs/**/*.spec.ts'
```

Comment:
Why are specs defined here but also in https://github.com/OutSystems/cordova-plugin-contacts/blob/ce9666ba82ecd4bc5974b18da32dc9d61385e1da/os-appium-tests/config/wdio.devicefarm.conf.js ?

Reply:
This is the default configuration that comes from the framework, and are used for local developments. For the AWS Device Farm, we are using the generated js files.

Reply:
Ah I see, thanks!